### PR TITLE
Add labels to issues on auto closing

### PR DESCRIPTION
**Pull request by stevenyomi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,9 +39,10 @@ jobs:
 
 ### Base
 
-| Name         | Description         |
-| ------------ | ------------------- |
-| `repo-token` | GitHub access token |
+| Name              | Description                                                |
+| ----------------- | ---------------------------------------------------------- |
+| `repo-token`      | GitHub access token                                        |
+| `duplicate-label` | Label to add on duplicate command and auto duplicate check |
```
Comment (arkon) [inline README.md:45]: Can you add this to the `test.yml` workflow in this repo too?

File: src/feature/auto-closer.ts [hunk 2]
```diff
@@ -94,6 +97,8 @@ export async function checkForAutoClose() {
         });
       }
 
+      await addLabels(client, issueMetadata, labels);
+
       await client.rest.issues.update({
```
Comment (arkon) [inline src/feature/auto-closer.ts:102]: Could technically do it as part of the `update` calls, but I guess it abstracts the need to read the input better.

File: src/util/issues.ts [hunk 3]
```diff
@@ -1,4 +1,12 @@
 import * as core from '@actions/core';
+import { GitHub } from '@actions/github/lib/utils';
+
+type GitHubClient = InstanceType<typeof GitHub>;
```
Comment (arkon) [inline src/util/issues.ts:4]: Should probably dedupe this declaration somewhere.

File: src/feature/auto-closer.ts (hunk 2 above)
Comment (stevenyomi) [reply src/feature/auto-closer.ts:102]: Yeah, the `update` call will override the labels, so we'll have to read original ones from the input and append the new ones. Also might risk losing those added by moderators during processing time.

File: src/util/issues.ts (hunk 3 above)
Comment (stevenyomi) [reply src/util/issues.ts]: Moved to `src/types.ts`. Should it be in `src/` or `src/util/`?

Comment (arkon) [reply src/util/issues.ts]: That's fine for now.

**Pull request merged**